src/modules/UI/selectors.js

```diff
@@ -202,6 +202,24 @@ export const calculateWalletFiatBalanceWithoutState = (wallet: GuiWallet, curren
   return formatNumber(fiatValue, { toFixed: 2 }) || '0'
 }
 
+export const calculateWalletFiatBalanceUsingDefaultIsoFiat = (
+  wallet: GuiWallet,
+  currencyCode: string,
+  settings: Object,
+  exchangeRates: { [string]: number }
+) => {
+  let fiatValue = 0 // default to zero if not calculable
+  const nativeBalance = wallet.nativeBalances[currencyCode]
+  if (!nativeBalance || nativeBalance === '0') return '0'
+  const denominations = settings[currencyCode].denominations
+  const exchangeDenomination = denominations.find(denomination => denomination.name === currencyCode)
+  if (!exchangeDenomination) return '0'
+  const nativeToExchangeRatio: string = exchangeDenomination.multiplier
+  const cryptoAmount: number = parseFloat(convertNativeToExchange(nativeToExchangeRatio)(nativeBalance))
+  fiatValue = convertCurrencyWithoutState(exchangeRates, currencyCode, settings.defaultIsoFiat, cryptoAmount)
+  return formatNumber(fiatValue, { toFixed: 2 }) || '0'
```

Comment:
Why format the number? Just return the number since it will never be displayed

Reply:
fixed